var/spack/repos/builtin/packages/py-pysam/package.py

```diff
@@ -30,12 +30,19 @@ class PyPysam(PythonPackage):
        sets."""
 
     homepage = "https://pypi.python.org/pypi/pysam"
-    url      = "https://pypi.io/packages/source/p/pysam/pysam-0.11.2.2.tar.gz"
+    url      = "https://github.com/pysam-developers/pysam/archive/v0.14.1.tar.gz"
 
+    version('0.14.1', 'ad88fa5bbcc0fdf4a936734691d9c9d1')
     version('0.13', 'a9b502dd1a7e6403e35e6972211688a2')
     version('0.11.2.2', '56230cd5f55b503845915b76c22d620a')
     version('0.7.7', 'eaf9f37cbccc5e2708754d045909c1a0')
 
     depends_on('py-setuptools', type='build')
     depends_on('py-cython@0.21:', type='build')
+    depends_on('curl')
     depends_on('bcftools')
+    depends_on('htslib')
+    depends_on('samtools')
+
+    # py-pysam@0.13 doesn't work with htslib@1.7
+    conflicts('htslib@1.7:', when='@:0.13')
```

Comment:
If you change this to:
```python
depends_on('htslib@:1.6', when='@:0.13')
```
it will fix the dependency version instead of crashing.

Reply:
Fixed.